### PR TITLE
TEIIDSB-154 adding an example of materialization with ispn

### PR DESCRIPTION
**Pull request by shawkins**

### CODE REVIEW

File: infinispan-materialization/Readme.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
+:toc:
+
+= Materialization with Infinispan Datagrid [[inifinspan-materialization]]
+
+This example shows how to utilize an Infinispan DataGrid on OpenShift as a materialization target.
+
+This will process vix data from link:https://datahub.io/core/finance-vix in the form of a csv file that is updated daily.
+
+== Prerequisites
```
Comment (rareddy) [inline infinispan-materialization/Readme.adoc:9]: Can we adjust this section to flow with the setup of minishift and Operator already in the documents?

File: infinispan-materialization/Readme.adoc [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+Deployment of the virtualization is *always* done using the Operator. In this example we'll use a yaml file to define a custom resource.
+
+Deploy the secret for accessing the DataGrid and the VDB with:
+
+----
+oc apply -f infinispan_teiid_secret.yaml
+oc create -f dv-mat.yaml
+----
+
+== Accessing the deployed Virtualization [vdb-access]
```
Comment (rareddy) [inline infinispan-materialization/Readme.adoc:94]: This section is also can be referenced from other documents that are present

File: infinispan-materialization/dv-mat.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    valueFrom:
+          secretKeyRef:
+            name: infinispan-teiid
+            key: password
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_URL
+    valueFrom:
+          secretKeyRef:
+            name: infinispan-teiid
+            key: url
+  - name: SPRING_TEIID_DATA_INFINISPAN_ISPN_TRANSACTIONMODE
```
Comment (rareddy) [inline infinispan-materialization/dv-mat.yaml:23]: are any of these default we can skip defining them?

File: infinispan-materialization/Readme.adoc (hunk 1 above)
Comment (shawkins) [reply infinispan-materialization/Readme.adoc:9]: I was not initially successful using minishift with the infinispan operator.  I think I had commented on that in the issue.

File: infinispan-materialization/Readme.adoc (hunk 2 above)
Comment (shawkins) [reply infinispan-materialization/Readme.adoc:94]: Yes, there were only a couple of updates and the addition of the odata stuff.  I'm fine if you want to make a more common reference for JDBC access.

File: infinispan-materialization/dv-mat.yaml (hunk 3 above)
Comment (shawkins) [reply infinispan-materialization/dv-mat.yaml:23]: None have a default.  The secret values will depend upon the infinispan setup.  Defaulting to non durable XA or NON_XA could be a breaking change for existing usage of the infinispan translator.  The closest you could argue for is to have a default cache name as "default" - which is not really an expectation of infinspan anymore (https://issues.redhat.com/browse/TEIID-5882) but we could just document / assume that.

**Review by rareddy — APPROVED** [review]

**Pull request merged**